profiles/desc/curl_default_ssl.desc

```diff
@@ -0,0 +1,11 @@
+# Copyright 1999-2020 Gentoo Foundation.
```

Comment:
We do 'Gentoo Authors' nowadays.

Reply:
I will update curl_ssl.desc while I'm at it since that is where that line came from.